courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>change title to "Exploring the natality dataset"</p><p> </p><p>modify text:</p><p>"Before we begin model development and training, we will explore the natality dataset. To do this, we will use BigQuery to explore the data and plot data visualizations using a Cloud AI Platform Notebook."</p><p> </p><p>"Try to complete the TODOs in this notebook first before reviewing the&nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/6004a8f7cba4433dd5694b43ec3b9073757afcb8/courses/machine_learning/deepdive2/structured/solutions/1a_explore_data_babyweight.ipynb" target="_blank">solution notebook</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>